### PR TITLE
Block type select

### PR DESCRIPTION
**Pull request by shivanshbindal9**
block_type
This adds react-select in block_type_select.

### CODE REVIEW

File: app/assets/stylesheets/modules/_timeline.styl [hunk 1]
```diff
@@ -131,11 +131,12 @@
 
 .block__block-type
   color $text_dk
-  font-size 14px
+  font-size 14  px
```
Comment (ragesoss) [inline app/assets/stylesheets/modules/_timeline.styl]: Is that a typo?

Comment (shivanshbindal9) [reply app/assets/stylesheets/modules/_timeline.styl]: Yeah, sorry I didn't notice it. 
I have fixed it.

**Comment by ragesoss** [comment]
This looks pretty  good — although Assignment still gets truncated.

There's one small regression: if you change the type, but then click 'Cancel', the input does not reset to the original value (while it does reset correctly in the current implementation).

**Comment by shivanshbindal9** [comment]
@ragesoss It is working fine on my system i.e. you click 'Cancel' the input does not reset.

**Comment by ragesoss** [comment]
If you click cancel, it 'should' reset.

**Comment by shivanshbindal9** [comment]
Sorry, my bad!! Can you please review it. @ragesoss

**Comment by shivanshbindal9** [comment]
Can you please review it @ragesoss !

**Pull request merged**